### ISSUE TITLE
AzureDevOps Invalid argument value. Parameter name:A description for a pull request must not be longer than 4000 characters.

### ISSUE THREAD

**Issue by aldodfm** [dependabot/dependabot-core#2747]
Maybe related with https://github.com/dependabot/dependabot-core/pull/2349
**Using:**
FROM dependabot/dependabot-core:0.125.2
gem 'dependabot-omnibus', '~> 0.125.2'

**Package manager/ecosystem**
javascript
**Manifest contents prior to update**
package.json
```json
{
  "name": "azure-devops-bug",
  "version": "0.1.0",
  "private": true,
  "engines": {
    "node": "^12.13 || ^14.15",
    "yarn": "^1.15.2"
  },
  "license": "UNLICENSED",
  "dependencies": {
    "history": "^4.10.1"
  },
  "scripts": {
    "build": "yarn lint:ts && yarn build:ts",
    "build:ts": "react-scripts build"
  }
}
```
**Updated dependency**
https://www.npmjs.com/package/history
**What you expected to see, versus what you actually saw**
Error already fixed/covered in code, but still failing:
```ruby
{"$id"=>"1", "innerException"=>nil, "message"=>"Invalid argument value.\r\nParameter name: A description for a pull request must not be longer than 4000 characters.", "typeName"=>"Microsoft.TeamFoundation.SourceControl.WebServer.InvalidArgumentValueException, Microsoft.TeamFoundation.SourceControl.WebServer", "typeKey"=>"InvalidArgumentValueException", "errorCode"=>0, "eventId"=>0}
```
**Images of the diff or a link to the PR, issue or logs**

**Comment by jurre** [comment]
Thanks, it is indeed related to #2349. It's a little confusing that the GitHub UI shows that PR as merged, but the commits were actually part of a different PR but where reverted in that same branch, so those changes never actually made it into the codebase.

I've opened up https://github.com/dependabot/dependabot-core/pull/2752 which I think should fix this, but I'll need a bit of time to add test coverage etc for it to verify this actually works.

**Comment by aldodfm** [comment]
thx @jurre , let's enjoy the update! :D

**Comment by jurre** [comment]
This should have been resolved by #2752

**Comment by ErisApps** [comment]
Pretty late to the party, but this issue doesn't seem to be resolve just yet.
While trying to update CommunityToolkit.Mvvm (https://github.com/CommunityToolkit/dotnet/releases) from 8.2.0 to 8.2.1,
I get the following error and the PR doesn't get created.
![image](https://github.com/dependabot/dependabot-core/assets/25928757/412f3630-a892-46a3-af0b-2480e930b340)
